core-web/apps/dotcms-ui/src/app/view/components/_common/dot-wizard/dot-wizard.component.ts

```diff
@@ -71,7 +72,10 @@ export class DotWizardComponent implements OnDestroy {
                 this.setDialogActions();
                 this.cd.detectChanges();
                 this.focusFistFormElement();
-            }, 1000);
+                setTimeout(() => {
+                    this.$stepsVisible.set(true);
```

Comment:
@hmoreras is there a way to avoid this timeout? may using afterNextRender?”

Reply:
I remove the timeouts, instead use the delay in rxjs, needed, but also important remove the manual detect changes.